folio-spring-system-user/pom.xml

```diff
@@ -19,6 +19,11 @@
       <version>${project.parent.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>util</artifactId>
+      <version>35.3.0</version>
```

Comment:
In parent pom `cql2pgjson.version` property exists, you can rename it, and use it here instead of using the version directly.